{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.1",
	"cveMetadata": {
		"cveId": "CVE-2020-35501",
		"assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
		"assignerShortName": "redhat",
		"state": "PUBLISHED",
		"datePublished": "2022-03-30T00:00:00",
		"dateUpdated": "2024-08-04T17:02:08.109Z",
		"dateReserved": "2020-12-17T00:00:00"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "53f830b8-0a3f-465b-8143-3b8a9948e749",
				"shortName": "redhat",
				"dateUpdated": "2022-10-07T00:00:00"
			},
			"descriptions": [
				{
					"lang": "en",
					"value": "A flaw was found in the Linux kernels implementation of audit rules, where a syscall can unexpectedly not be correctly not be logged by the audit subsystem"
				}
			],
			"affected": [
				{
					"vendor": "n/a",
					"product": "kernel",
					"versions": [
						{
							"status": "affected",
							"version": "affecting all versions up to kernel 5.17"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "CWE",
							"lang": "en",
							"description": "CWE-863",
							"cweId": "CWE-863"
						}
					]
				}
			],
			"references": [
				{
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1908577"
				}
			]
		},
		"adp": [
			{
				"providerMetadata": {
					"orgID": "af854a3a-2127-422b-91ae-364da2661108",
					"shortName": "CVE",
					"dateUpdated": "2024-08-04T17:02:08.109Z"
				},
				"title": "CVE Program Container",
				"references": [
					{
						"tags": [
							"x_transferred"
						],
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1908577"
					}
				]
			}
		]
	}
}
